flake.lock

```diff
@@ -103,6 +100,20 @@
         "type": "github"
       }
     },
+    "nixpkgs_2": {
+      "locked": {
+        "lastModified": 1660817299,
+        "narHash": "sha256-0QZE5FYLOyTEpKobPk32gxAQBtYs/nAfGx3PK+x6bXI=",
+        "owner": "NixOS",
+        "repo": "nixpkgs",
+        "rev": "b42e50fe36242b1b205a7d501b7911d698218086",
+        "type": "github"
+      },
+      "original": {
+        "id": "nixpkgs",
+        "type": "indirect"
+      }
+    },
```

Comment:
What's this change about?

Reply:
not sure, was made automatically when I did `nix develop .`